src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java

```diff
@@ -101,6 +101,49 @@ public void findByZipFileName_should_find_envelops_in_db() {
         assertThat(resultForX).hasSize(0);
     }
 
+    @Test
+    public void should_find_envelope_when_ccdid_matches() {
+        // given
+        Envelope e1 = envelope("A.zip", "X", Status.CREATED);
+        e1.setCcdId("97586937");
+        dbHas(e1);
+
+        // when
+        List<Envelope> result = repo.findByCcdId("97586937");
+        // then
+        assertThat(result).hasSize(1);
```

Comment:
do not we need to assert envelope fields?

Reply:
Done, was thinking of doing this in serviceTest class